client/consensus/manual-seal/src/lib.rs

```diff
@@ -303,6 +321,42 @@ pub async fn run_instant_seal_and_finalize<B, BI, CB, E, C, TP, SC, CIDP, P>(
 	.await
 }
 
+pub async fn run_delayed_finalize<B, CB, C, S>(
```

Comment:
Some docs on how to use it. A "real" example would be  really nice, but at least saying that you don't set `finalize` in the command stream etc.